src/screens/SettingsScreen.js

```diff
@@ -54,6 +53,18 @@ export const SettingsScreen = ({ navigation }) => {
     }, 500);
   };
 
+  const onPress = async (selectedListType, queryType) =>
```

Comment:
I think a more specific name than `onPress` inside of a component with many things that can be pressed/touched would be appropriate

Reply:
agree

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/129/commits/552c906697f61fb7cb7d322ae6366ce47f5de9e1